Client/Frontend/Brave Rewards/Ads/AdsNotificationHandler.swift

```diff
@@ -72,7 +72,7 @@ public class AdsNotificationHandler: BraveAdsNotificationHandler {
         self.ads.reportAdNotificationEvent(notification.uuid, eventType: .timedOut)
       case .disliked:
         self.ads.reportAdNotificationEvent(notification.uuid, eventType: .dismissed)
-        self.ads.toggleThumbsDown(forAd: notification.uuid)
+          self.ads.toggleThumbsDown(forAd: notification.uuid, advertiserId: notification.advertiserID)
```

Comment:
nit: unnecessary space

Reply:
oops, i will wait for CI to pass, then update and merge it without subsequent ci pass